### DISCUSSION TITLE
FCFeynman functions are not available

### DISCUSSION THREAD

**Discussion by marcelogcampos** [FeynCalc/feyncalc#229] [Q&A] [answered]
Hi Vlad,

A new issue. The FCFeynman functions are not available, among them are: FCFeynmanFindDivergence, FCFeynmanParametrize, FCFeynmanPrepare, FCFeynmanParameterJoin. Some of the FCLoop are not too, as FCLoopAddEdgeTags, FCLoopGraphPlot, FCLoopIntegraltoGraph, and FCiteratedIntegralEvaluate, FCiteratedIntegralSimplify.
When I type them in Mathematica, they do not show up among the options and their code keep in blue. 

Could help me?

My FeynCalc version is 9.3.1, FeynArts 3.11 and Mathematica 10.

Fraternally,
marcelo

**Comment by vsht** [comment, marked as answer]
Hi,

those new functions are available only in the dev version,
not in the stable one.

Cheers,
Vlad

Am 06.09.23 um 02:44 schrieb marcelogcampos:
